astropy/table/tests/test_index.py

```diff
@@ -4,18 +4,30 @@
 import numpy as np
 
 from .test_table import SetupData
-from ..bst import BST, FastRBT
+from ..bst import BST, FastRBT, FastBST
 from ..sorted_array import SortedArray
 from ..table import QTable, Row
 from ... import units as u
 from ...time import Time
 from ..column import BaseColumn
 from ...extern.six.moves import range
 
-
-@pytest.fixture(params=[BST, FastRBT, SortedArray])
-def engine(request):
-    return request.param
+try:
+    import bintrees
+except ImportError:
+    _HAS_BINTREES = False
```

Comment:
Not a big deal but typically (at least from my experience) these `HAS_` vars are not prefixed with `_`.